### ISSUE TITLE
Allow fallback customization for non-project files?

### ISSUE THREAD

**Issue by lukoshkin** [ahmedkhalf/project.nvim#70]
Can we not change the directory for non-project files?
Being in directory ~/.config/nvim/ and opening one of its files, changes $CWD to $HOME. When I want to switch to another file whose prefix coincides with ~/.config/nvim/, I can't use telescope find_files, since it will use within $HOME dir. This is quite annoying.

May we consider sth similar to PR #62

**Comment by ahmedkhalf** [comment]
Does PR #62 fix your issue, or is it still present?

**Comment by lukoshkin** [comment]
I guess it's still present.

**Comment by ahmedkhalf** [comment]
so my guess is that you have some sort of file in your home directory that matches one of the `patterns`. You can set `silent_chdir = false` to debug.

**Comment by lukoshkin** [comment]
Hi, Ahmed!
Indeed, I had package.json in my HOME. (probably, I installed it ocassionally with npm, but never mind). Now when I open from a directory, CWD is not changed to HOME and is set to that directory. So I can say that everything works as expected for opening just one file.

However, what is weird (IMO) is that when opening one project and then another (without closing Neovim), my CWD is set to the root of the first project and remains unchanged when switching windows/buffers. It does not change even when calling ProjectRoot. The latter bothers me the most.

I could open a new issue devoted to the problem I've just stated if you told me that it is not sth obvious that I didn't configure
